src/views/homepage/components/featured-content-grid/featured-card.module.css

```diff
@@ -15,15 +14,12 @@
 	}
 }
 
-.learnMoreCta {
-	/* composition */
-	composes: hds-font-weight-medium from global;
-	composes: hds-typography-body-300 from global;
-
-	/* properties */
-	align-items: center;
+.learnMoreCtaContainer {
 	align-self: flex-end;
-	display: flex;
+}
+
+.learnMoreCta {
+	color: var(--token-color-palette-neutral-0);
```

Comment:
🗒️ All of these changes were necessary after changing the markup to leverage `StandaloneLinkContents`.